packages/create/templates/readme.hbs

```diff
@@ -117,6 +117,15 @@ can set `dbConnectionOptions.synchronize` to `true`. This will cause the databas
 on each start, removing the need for migration files. Note that this is **not** recommended once you have production
 data that you cannot lose.
 
+## Troubleshooting
+
+### Error: Could not load the "sharp" module using the \[OS\]-x\[Architecture\] runtime when running Vendure server.
+
+- Make sure your Node version is ^18.17.0 || ^20.3.0 || >=21.0.0 to support the Sharp library.
+- Make sure your package manager is up to date.
+- **Not recommended**: if none of the above helps to resolve the issue, install sharp specifying your machines OS and Architecture. For example: `pnpm install sharp --config.platform=linux --config.architecture=x64` or `npm install sharp --os linux --cpu x64`
+
 ---
 
 You can also run any pending migrations manually, without starting the server via the "vendure migrate" command.
```

Comment:
This paragraph should be part of the "migrations" section

Reply:
Oops sorry! Fixed now